### PR TITLE
[v6r15] Never throw an exception when sending failover accounting request

### PR DESCRIPTION
**Pull request by chaen**
Fix for https://github.com/DIRACGrid/DIRAC/issues/2416

### CODE REVIEW

File: AccountingSystem/Client/DataStoreClient.py [hunk 1]
```diff
@@ -137,14 +137,18 @@ def ping( self ):
...
+    forwardDISETOp = Operation()
+    forwardDISETOp.Type = "ForwardDISET"
+    forwardDISETOp.Arguments = DEncode.encode( rpcStub )
+    request.addOperation( forwardDISETOp )
+
+    return ReqClient().putRequest( request )
+
+  # We catch all the exceptions, because it should never crash
+  except Exception as e:  # pylint: disable=broad-except
+    return S_ERROR( "Exception sending accounting failover request: %s" % repr( e ) )
```
Comment (atsareg) [inline AccountingSystem/Client/DataStoreClient.py]: New style S_ERROR ?

Comment (chaen) [inline AccountingSystem/Client/DataStoreClient.py]: I thought of it, but I don't know what to put in there when we catch a generic exception. 
I can put the generic place holder for the RMS error ?

Comment (atsareg) [inline AccountingSystem/Client/DataStoreClient.py]: Yes, ERMSUKN.

**Comment by atsareg** [comment]
Review OK

**Pull request merged**